### PR TITLE
Lay the groundwork for localization

### PR DESCRIPTION
**Pull request by slice**
Using Project Fluent for localization support, implement general integration in the bot.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -159,12 +160,22 @@ def redis(self):
...
+    def lang(self):
+        return self.get_cog("Lang").fluent
+
     @property
     def sprites(self):
         return self.get_cog("Sprites")
 
     # Other stuff
 
+    def _(self, message_id: str, **kwargs: dict[str, typing.Any] | None) -> str:
```
Comment (oliver-ni) [inline bot.py:173]: I don't think this is the correct typing for `**kwargs`

File: cogs/help.py [hunk 2]
```diff
@@ -6,38 +6,87 @@
...
-        super().__init__(command_attrs={"help": "Show help about the bot, a command, or a category."})
-
     async def on_help_command_error(self, ctx, error):
         if isinstance(error, commands.CommandInvokeError):
             await ctx.send(str(error.original))
 
-    def make_page_embed(self, commands, title="Pokétwo Help", description=None):
+    def resolve_command_help(self, command: commands.Command) -> str:
+        """Resolve a command's help message using Fluent, falling back to the
+        command's docstring."""
```
Comment (oliver-ni) [inline cogs/help.py:15]: ending `"""` should be on a new line I think

File: helpers/context.py [hunk 3]
```diff
@@ -144,6 +144,20 @@ def __init__(self, *args, **kwargs):
...
+        known prefix value.
+        """
+        # HACK: Reach into the Lang cog and update its last known prefix value,
+        # so Fluent localizations that use the `COMMAND` function format
+        # correctly.
+        #
+        # This could be done properly if terms weren't forced to be
+        # parameterized with solely literals; that prevents us from passing it
+        # the usual way.
+        self.bot.get_cog("Lang")._last_known_prefix = self.clean_prefix
```
Comment (oliver-ni) [inline helpers/context.py:158]: talking about this in discord

**Review by oliver-ni — CHANGES_REQUESTED** [review]

File: bot.py (hunk 1 above)
Comment (slice) [reply bot.py:173]: It should be according to `fluent.runtime` source, unless I'm missing something?

```py
def format_value(self, msg_id: str, args: Union[Dict[str, Any], None] = None) -> str:
```

Comment (slice) [reply bot.py:173]: There's wrappers for Fluent types in `fluent.runtime.types`, like `FluentType` and `FluentNumber`, but those seem to be data wrappers that are used internally (?) by the library.

**Comment by slice** [comment]
(Concerning that commit I just pushed, we're still using `cog.qualified_name` inside of `send_bot_help` because it ends up calling `get_cog` after sorting.)

**Pull request merged**